kubeflow/core/kubeform_spawner.py

```diff
@@ -102,6 +112,7 @@ def extra_resource_limits(self):
 repoName = os.environ.get('REPO_NAME')
 c.JupyterHub.spawner_class = KubeFormSpawner
 c.KubeSpawner.singleuser_image_spec = '{0}/{1}/tensorflow-notebook'.format(registry, repoName)
+c.KubeSpawner.image_spec = '{0}/{1}/tensorflow-notebook'.format(registry, repoName)
```

Comment:
What's this for?

Reply:
I should've clarified this. Some fields have been deprecated in `KubeSpawner` class[1]. I've added the new field names so that this config file is forward compatible with future jupyterhub versions.

[1] https://github.com/jupyterhub/kubespawner/blob/master/kubespawner/spawner.py#L986